cyfra-core/src/main/scala/io/computenode/cyfra/core/layout/LayoutStruct.scala

```diff
@@ -60,11 +60,11 @@ object LayoutStruct:
             ftype match
               case '[type tg <: GBuffer[?]; tg] =>
                 '{
-                  BufferRef[t](${ Expr(i) }, ${ tag.asExprOf[Tag[t]] })(using summon[Tag[t]], ${ fromExpr.asExprOf[FromExpr[t]] })
+                  BufferRef[t](${ Expr(i) }, ${ tag.asExprOf[Tag[t]] })(using ${ tag.asExprOf[Tag[t]] }, ${ fromExpr.asExprOf[FromExpr[t]] })
```

Comment:
I "fixed" the bug here, but I'm not sure if it's correct... it just compiles now.